### PR TITLE
Nd implement browser router

### LINKED ISSUE #13: Route to SavedPlaylists in App

### LINKED ISSUE #14: Render homepage components in App

### PR DESCRIPTION
**Pull request by npdarrington**
### What’s this PR do? 

- Added Browser Router to Index.tsx
- Setup Routes inside Router in App.tsx file 
- Conditionally render a path in PlaylistContainer (seeing if this will allow us to only have 1 playlist container that can render each individual component)
- Added some structure to see conditional renders for specific Apps
- Render Header componet in App.
 
### Where should the reviewer start?  

- Git pull this branch
 
### How should this be manually tested?  

- Run `npm install` to install any dependencies if this is the first pull
- Run `npm start` and navigate to `localhost:3000`
- On first render, you should see Genrefy, Genre, Custom Playlist and Generated Playlist title render
- Navigate to `localhost:3000/saved-playlist` to see Genrefy and Saved Playlist title render
 
### Any background context you want to provide?  

- N/A
 
### What are the relevant tickets?  

- closes #13 
- closes #14
 
### Screenshots (if appropriate)  
- N/A
 
### Questions: 
- N/A

### CODE REVIEW

File: src/Header/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+function Header() {
```
Comment (bretm9) [inline src/Header/Header.tsx:3]: Thanks for populating Header as a basic component!

File: src/PlaylistContainer/PlaylistContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  if (playlistType === 'saved-playlist') {
+    return <h1>Saved Playlist</h1>
+  }
+  if (playlistType === 'generated-playlist') {
+    return <h1>Generated Playlist</h1>
+  }
+  if (playlistType === 'custom-playlist') {
+    return <h1>Custom Playlist</h1>
+  }
+}
```
Comment (DougieDev) [inline src/PlaylistContainer/PlaylistContainer.tsx:13]: Love this. Keeping the return small using destructuring for conditional rendering is much nicer than having a large potentially complex conditional render in our functional component. Big props. 👌

**Review by DougieDev — APPROVED** [review]

File: src/PlaylistContainer/PlaylistContainer.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+const displayPlaylist = (playlistType: string) => {
+  if (playlistType === 'saved-playlist') {
+    return <h1>Saved Playlist</h1>
+  }
+  if (playlistType === 'generated-playlist') {
+    return <h1>Generated Playlist</h1>
+  }
+  if (playlistType === 'custom-playlist') {
+    return <h1>Custom Playlist</h1>
```
Comment (bretm9) [inline src/PlaylistContainer/PlaylistContainer.tsx:11]: This seems like a good way of avoiding conditional rendering without our `return`

File: src/PlaylistContainer/PlaylistContainer.tsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  }
+}
+
+function PlaylistContainer(props: any) {
+  return (
+    <section className='playlist-container'>
+      { displayPlaylist(props.playlistType) }
```
Comment (bretm9) [inline src/PlaylistContainer/PlaylistContainer.tsx:18]: In the future we should type props.playlistType

**Review by bretm9 — APPROVED** [review]
Routes look like they're functioning correctly. I think our app is all setup and we can begin digging into our components functionality!

**Pull request merged**